app/views/components/_share-links.html.erb

```diff
@@ -12,15 +12,25 @@
         target: "_blank",
         rel: "noopener noreferrer",
         class: "app-c-share-links__link js-share-facebook" do %>
-        <span class="app-c-share-links__link__icon app-c-share-links__link__icon--facebook"></span><span class="visually-hidden">Share on </span>Facebook<% end %>
+        <span class="app-c-share-links__link__icon app-c-share-links__link__icon--facebook">
+          <svg xmlns="http://www.w3.org/2000/svg" width="32" height="32">
+            <path fill="currentColor" d="M31.006 0H.993A.997.997 0 0 0 0 .993v30.014c0 .55.452.993.993.993h30.013a.998.998 0 0 0 .994-.993V.993A.999.999 0 0 0 31.006 0z"/>
+            <path fill="#FFF" d="M17.892 10.751h1.787V8.009L17.216 8c-2.73 0-3.352 2.045-3.352 3.353v1.828h-1.581v2.824h1.581V24h3.322v-7.995h2.242l.291-2.824h-2.533V11.52c.001-.623.415-.769.706-.769z"/>
+          </svg>
+        </span><span class="visually-hidden">Share on </span>Facebook<% end %>
     <% end %>
 
     <% if twitter_href %>
       <%= link_to twitter_href,
         target: "_blank",
         rel: "noopener noreferrer",
         class: "app-c-share-links__link js-share-twitter" do %>
-        <span class="app-c-share-links__link__icon app-c-share-links__link__icon--twitter"></span><span class="visually-hidden">Share on </span>Twitter<% end %>
+        <span class="app-c-share-links__link__icon app-c-share-links__link__icon--twitter">
```

Comment:
Nit: we don't seem to be using the --twitter and --facebook classes anymore, can they be removed?